src/render/GameMenuScreen.java

```diff
@@ -31,7 +32,7 @@ public void blit(double screenWidth, double screenHeight)
         for (GuiElement item : selectable) {
             int xPos = ((int)screenWidth - item.getWidth()) / 2;
```

Comment:
Should this be:
            int xPos = (int)((screenWidth - item.getWidth() / 2);
instead?

Reply:
On further inspection, this change is actually fine. I was thinking of the two-column logic from the other screen.